wasmcloud-chat/actors/rooms/src/kv.rs

```diff
@@ -0,0 +1,58 @@
+use serde::{Deserialize, Serialize};
+use wapc_guest::prelude::*;
+use wasmcloud_actor_keyvalue as keyvalue;
+
+#[derive(Serialize, Deserialize, PartialEq, Clone, Debug)]
+pub(crate) struct RoomData {
+    pub id: String,
+    pub description: String,
+}
```

Comment:
Shouldn't this just be the `Room` struct from the interface, and then when you set the value later the member # should be 0?

Reply:
that'd be "tier bleeding". the room struct from the interface is the _externally facing_ data, the `RoomData` struct is the internal data as stored in redis. It's only a coincidence that they have the same fields now. By keeping them separate now, I can change the internal redis representation without having to bust the public API.